src/nlp/dataset.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pandas as pd
+import configparser
+from typing import NoReturn, Tuple
+from utils import preprocess_title
+from torch.utils.data import Dataset
+from transformers import BertTokenizer
+
+CONFIG = configparser.ConfigParser()
+CONFIG.read("nlp_config.ini")
```

Comment:
Use config.py or write a function to parse the .ini file

Reply:
Added parsing function in utils.py